File: backend/src/Devsu.Application/Dtos/Accounts/GetAccount.cs
namespace Devsu.Application.Dtos.Accounts;

public class GetAccount
{
    public Guid Id { get; set; }
    public string? AccountNumber { get; set; }
    public string? AccountType { get; set; }
    public decimal InitialBalance { get; set; }
    public decimal CurrentBalance { get; set; }
    public decimal DailyDebitLimit { get; set; } 
    public GetUser? User { get; set; }
    public Guid UserId { get; set; }
}

File: front-angular/src/app/accounts/interfaces/get-account.ts
import { GetUser } from "../../users/interfaces/get-user";

export interface GetAccount {
  id: string;
  accountNumber: string;
  accountType: "ahorros" | "corriente";
  initialBalance: number;
  currentBalance: number;
  dailyDebitLimit: number;
  user?: GetUser;
  userId: string;
}
